kafka/src/main/java/io/specmesh/kafka/provision/schema/SchemaProvisioner.java

```diff
@@ -175,8 +202,17 @@ private static List<Schema> requiredSchemas(
     private static String resolveSubjectName(
             final String topicName,
             final Collection<ParsedSchema> schemas,
-            final SchemaInfo schemaInfo) {
-        final var lookup = schemaInfo.schemaLookupStrategy();
+            final SchemaInfo schemaInfo,
+            final String partName) {
+        final String lookup =
```

Comment:
Not sure if you want to default this to something? e.g. `TopicNameStrategy`?

Reply:
default to TopicNameStrategy